### ISSUE TITLE
Chapter 10: spelling and mark-up errors, extra explanation

### ISSUE THREAD

**Issue by Dries-B** [microbiome/OMA#203]
Dear authors,

Thank you for this hands-on guide to microbiome analysis!

When reading, I noticed that Chapter 10 contains quite some spelling errors and mark-up errors. Moreover, for me it seems that some of the analyses could use more explanation; for example, the determination of optimal `Number_clusters` by `NbClust()` in chunk `hclust3`.

**Comment by Dries-B** [comment]
I could help with correcting the spelling mistakes, but I think that should probably happen simultaneously with adding more explanatory text. And someone more knowledgeable about clustering would be more suitable for that task.

**Comment by antagomir** [comment]
Many thanks! I agree and indeed this is still beta version, including multiple contributors and variation and some of the chapters are less finished than others. We are hoping to find the time to unify and polish the text more very soon. If you can contribute some useful improvements via Pull Request, even if it is to typos or language, this will help and will be reflected on the contributor list.

**Comment by antagomir** [comment]
@BananaCancer and @TuomasBorman perhaps you could check & close this one?

**Comment by BananaCancer** [comment]
I'm updating the chapter 10 in the PR #295 if you want to check.

**Comment by antagomir** [comment]
Yes, will check there. Just ensure that this issue is handled and then closed together with the main PR.
